doc/migrating-configs.md

```diff
@@ -170,27 +170,6 @@ The `ignition` section has gained a new section named `security`, which can be u
 }
 ```
 
-### networkd dropins
```

Comment:
I'm not sure we should be rewriting history like this.  Instead, the migration instructions to version 3 should explain converting networkd units/dropins to regular files.

Reply:
Dropped this change and opened up #705